bp_common/src/include/bp_common_csr_defines.vh

```diff
@@ -485,35 +485,20 @@ typedef struct packed
 
 typedef struct packed
 {
-  logic mei;
```

Comment:
Shouldn't this be only m_* and not s_*? setting sti means delegating s-mode timer interrupts to u-mode, no?

Reply:
No. It means sei, sti, ssi will trap to S mode instead of M mode.
We don't need the m* ones because we don't want to delegate mei, mti, msi to S mode.

Reply:
The X in Xei, Xti, Xsi doesn't have to do anything with the mode we are running that. It simply means that `Xip & Xie == 1`

Reply:
For delegating to U, we need the sideleg CSR.